lib/AppInfo/Application.php

```diff
@@ -218,4 +224,26 @@ protected function registerCollaborationResources(IProviderManager $resourceMana
 			Util::addScript('deck', 'deck-collections');
 		});
 	}
+
+	protected function registerSessionListener(IEventDispatcher $eventDispatcher, SessionService $sessionService): void {
+		$container = $this->getContainer();
+		try {
+			$queue = $container->get(IQueue::class);
+		} catch (\Exception $e) {
+			// most likely notify_push is not installed.
+			return;
+		}
+		
+		$eventDispatcher->addListener(SessionCreatedEvent::class, function (SessionCreatedEvent $event) use ($sessionService, $queue) {
+			$sessionService->notifyAllSessions($queue, $event->getBoardId(), "DeckBoardUpdate", $event->getUserId(), [
```

Comment:
Would be nice to use constants for the events like DeckBoardUpdate and maybe stick to snake case as that seems to be common with https://github.com/nextcloud/notify_push/blob/main/DEVELOPING.md#sending-custom-events: `deck_board_update`